shorebird_code_push/example/lib/main.dart

```diff
@@ -1,125 +1,108 @@
 import 'package:flutter/material.dart';
 
+import 'package:shorebird_code_push/shorebird_code_push.dart';
+
+final ShorebirdCodePush shorebirdCodePush = ShorebirdCodePush();
```

Comment:
```suggestion
final shorebirdCodePush = ShorebirdCodePush();
```

Would be nice to use the same analysis options we use in the shorebird repo for consistency.